shakenfist/daemons/daemon.py

```diff
@@ -48,6 +49,13 @@ def __init__(self, name):
         log, _ = logutil.setup(name)
         set_log_level(log, name)
 
+        self.running = True
+        signal.signal(signal.SIGTERM, self.exit_gracefully)
+
+    def exit_gracefully(self):
```

Comment:
`The handler is called with two arguments: the signal number and the current stack frame `

Reply:
Oh cool. Thanks!